lib/python-beta/src/bailo/helper/model.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import os
+from typing import Any
+
+from bailo.core import Client, ModelVisibility
+from bailo.helper.utils import retrieve_model
+
+
+class Model:
+    def __init__(
+        self,
+        client: Client,
+        name: str,
+        description: str,
+        visibility: ModelVisibility | None = ModelVisibility.Public,
+        model_card: dict[str, Any] | None = None
+    ):
+        self.client = client
+        self.name, self.description, self.visibility = name, description, visibility
+        self.model_id = None
+        self.model_version = None
+        self.model_card_version = None
+        self.model_card_schema = 'minimal-general-v10-beta'
+        self.__model_card_original = model_card
+        self.__model_card = model_card
+        self.__local_dir = f"./temp_{name.strip(' ')}"  # property tbc
+        self.__mc_change = 0
+        self.__file_change = 0  # property tbc
+
+    @classmethod
+    def from_id(cls, client: Client, model_id: str):
+        name, description, visibility, model_card, model_card_version, model_card_schema = retrieve_model(client=client, model_id=model_id)
+        model = cls(
+            client=client,
+            name=name,
+            description=description,
+            visibility=visibility,
+            model_card=model_card
+        )
+        model.model_id = model_id
+        model.model_card_version = model_card_version
+        model.model_card_schema = model_card_schema
+
+        return model
+
+    def refresh(self):
+        self.name, self.description, self.visibility, self.__model_card, self.model_card_version, self.model_card_schema = retrieve_model(client=self.client, model_id=self.model_id)
+
+    def publish(self):
```

Comment:
Instead of a `.publish()` function, I think it would be better to have a static method called `create` or equivalent.  I believe this has been done on the `Releases`?

It'll make things like the 'from_id' function above cleaner as well.